lib/worker/batcher.ex

```diff
@@ -550,26 +550,26 @@ defmodule BorsNG.Worker.Batcher do
     patches = id
     |> Patch.all_for_batch()
     |> Repo.all()
-    send_status(repo_conn, patches, message)
+    send_status(repo_conn, id, patches, message)
     unless is_nil commit do
       {msg, status} = Batcher.Message.generate_status(message)
       repo_conn
       |> GitHub.post_commit_status!({
         commit,
         status,
         msg,
-        project_url(Endpoint, :log, project_id)})
+        project_url(Endpoint, :log, project_id) <> "#batch-#{id}"})
```

Comment:
afaict there is no way to put attach a URL fragment inside the URL helper itself.

Reply:
I'm not too worried about this. We know that the batch ID is a self-generated integer, so it's safe to stick it into a URL (if it were computed from user input, I'd be more worried).